src/server/routers/account.ts

```diff
@@ -10,7 +10,34 @@ export const accountRouter = t.router({
         },
       },
     });
+    return accounts
+  })
```

Comment:
when creating new functions / endpoints , you can just add them under same router : accountRouter

Reply:
Okay i will change that